src/Web/Controllers/AccountController.cs

```diff
@@ -229,14 +235,19 @@ public async Task<IActionResult> Register(RegisterViewModel model, string return
             return await RegisterUser(model, returnUrl);
         }
 
-        private async Task<IActionResult> CallCustomAssessmentApi(CustomAssessmentViewModel model, string returnUrl, string envId)
```

Comment:
what was returnUrl previously used for?

Reply:
It wasn't used, actually.  I assume it was there from copy/paste.